index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Website</title>
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+</head>
+<body>
+<nav class="bg-indigo-800 flex flex-row text-white h-1/6 justify-center space-x-6 ">
+    <a class="hover:text-gray-900 shadow" href="#">Home</a>
+    <a class="hover:text-gray-900" href="contact.html">Contact</a>
+    <a class="hover:text-gray-900" href="blog.html">Blog</a>
+    </nav>
+<main class=" h-screen  bg-black flex flex-row">
+    <section class="h-auto w-11/12 m-8 bg-yellow-300 flex flex-row
+    justify-evenly justify-center items-end">
```

Comment:
you're using both `justify-evenly` and `justify-center`. One of them is overriding the other, decide which one you want and remove the other one

Reply:
I will keep justify-evenly thanks